api.js

```diff
@@ -675,9 +675,10 @@ API.prototype.getBrowserTimingHeader = function getBrowserTimingHeader(options)
 
   // set nonce attribute if passed in options
   const nonce = options && options.nonce ? 'nonce="' + options.nonce + '"' : ''
```

Comment:
Can you update the jsdoc for this function?  We publish these docs to http://newrelic.github.io/node-newrelic/docs/

Reply:
Of course, I'll do 
Thanks :+1: